geoblacklight/Gemfile

```diff
@@ -0,0 +1,65 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.0.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
```

Comment:
consider using the postgres container rather than sqlite - it does not perform when there is more than one user